### DISCUSSION TITLE
Add one-key fix to windows standby draining battery issue

### DISCUSSION THREAD

**Discussion by TachibanaOchiba** [ChrisTitusTech/winutil#494] [Ideas]
- Per LinusTechtip's video, modern windows laptops usually drain battery fast in standby mode. This is possibly caused by connected standby feature sometimes fails to register that power has been disconnected. This bug has been there for a long time and causes bad user experience.

- Connected standby is enabled by default in fresh windows 10/11 installs. However, users can disable it by disabling group policy ’ Allow network connectivity during connected-standby (plugged in)‘ and ‘ Allow network connectivity during connected-standby (using battery)’.

- Maybe add a tickbox to do this in the tweak tab so that everyone has a laptop that does not heat itself to death in carrying bags?

LTT video: https://www.youtube.com/watch?v=OHKKcd3sx2c&t
Default settings of connected standby:![Snipaste_2022-12-06_00-19-42](https://user-images.githubusercontent.com/48231576/205687588-539765cd-af17-4939-8013-efcc17387152.png)

**Comment by DeveloperDurp** [comment]
Once #576 gets merged I will investigate this and attempt to implement if possible.
